catroid/src/main/java/org/catrobat/catroid/ui/adapter/BrickAdapter.java

```diff
@@ -976,6 +959,9 @@ public void onClick(final View view) {
 				&& !(brickList.get(itemPosition) instanceof ScriptBrick)) {
 			items.add(context.getText(R.string.brick_context_dialog_move_brick));
 		}
+		if ((brickList.get(itemPosition) instanceof UserBrick)) {
```

Comment:
This can actually be done with several if-branches.. e.g. the NestingBrick-check can also be an else-if, as it can't be either a ScriptBrick nor a UserBrick --> so I finally added two else-if's here